spec/lib/nacre/journal/entry_collection_spec.rb

```diff
@@ -0,0 +1,48 @@
+require "spec_helper"
+
+describe Nacre::Journal::EntryCollection do
+  let(:params) do
+    [
+      {
+        nominal_code: "1001",
+        change: {
+            value: "99.99"
```

Comment:
Use 2 spaces for indentation in a hash, relative to the start of the line where the left curly brace is.